MusicBandSimulator/MusicBandSimulator/BandMember.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MusicBandSimulator
+{
+    /// <summary>
+    /// Об'єкт - Учасник Гурту. Містить конструктор для створення нового учасника з ім'ям та музичним інструментом. Таком містить метод, який дозволяє учаснику гурту грати на своєму інструменті.
+    /// </summary>
+    internal class BandMember
+    {
+        public string nameOfMember { get; set; }
+        public IMusicInstrument memberInstrument { get; set; }
+
+        public BandMember(string nameOfMember, IMusicInstrument memberInstrument)
+        {
+            this.nameOfMember = nameOfMember;
+            this.memberInstrument = memberInstrument;
+        }
+
+        public void BandMemberPerform()
```

Comment:
```suggestion
        public void Perform()
```

Reply:
done but dont understand - for why? isn't it better to have the name more clear to understand? i have already name "Perform" in MusicBand class... and it would be hard to understand, where "perform" applies to MusicBand class and where - to BandMember